sites/friday/src/get-started/here/make-changes.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Make changes to your project
+sidebarTitle: Revisions
+weight: 50
+description: How to work on a daily basis with an {{% vendor/name %}} project?
+---
+
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+It's this component of Upsun - preview environments - that you'll come to recognize as a key feature.
+
+In your project, the `main` branch (or `master` if you change it during project creation) always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+1. Create a new environment (a Git branch) to make changes without impacting production:
+{{< codetabs >}}
++++
+title={{% vendor/name %}} Git repository
++++
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} branch feat-a
```

Comment:
given we're trying to juggle integrations and non-integrations at this point, is there any advantage to instructing the user to use the our cli to create the branch over just using git? By using our CLI we need to include additional steps for the integration path.